closeaccountingperiod.php

```diff
@@ -394,3 +394,60 @@ function closeaccountingperiod_civicrm_alterSettingsMetaData(&$settingsMetadata,
     'help_text' => '',
   );
 }
+
+/**
+ * Implementation of hook_civicrm_alterReportVar
+ */
+function closeaccountingperiod_civicrm_alterReportVar($varType, &$var, &$object) {
```

Comment:
@Edzelopez Trial Balance is part of this extension. Any reason why this logic should be implemented as part of hook rather directly in https://github.com/JMAConsulting/biz.jmaconsulting.closeaccountingperiod/blob/master/CRM/CloseAccountingPeriod/Form/Report/TrialBalance.php